portfolio/src/main/webapp/style.css

```diff
@@ -319,10 +319,23 @@ input[type=submit]:hover,
 
 #login-status {
   background-color: white;
-  top: 0;
-  padding: 20px;
+  column-gap: 5px;
+  display: grid;
+  grid-template-columns: auto auto;
+  padding: 10px;
   position: fixed;
   right: 0;
+  top: 0;
+}
+
+.comment-width-wrapper {
```

Comment:
SInce your css file is pretty long, it might be helpful to divide it into sections by adding comments (the style guide suggests it here: https://google.github.io/styleguide/htmlcssguide.html#Section_Comments)